<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(18, 130, 119)</string>
  </property>
  <widget class="QPushButton" name="add_user_food">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>40</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>Добавить свою еду</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_food">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>Название продукта, 0, 0, 0, 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>70</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>Название, кол-во ккал, кол-во б, ж, у </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>(на 100 г продукта)</string>
   </property>
  </widget>
  <widget class="QListWidget" name="food_list">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>220</y>
     <width>271</width>
     <height>192</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(185, 196, 201)</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
   </property>
  </widget>
  <widget class="QPushButton" name="accept">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>170</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>Подтвердить</string>
   </property>
  </widget>
  <widget class="QLabel" name="dinner_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>Ужин: 0 ккал</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_to_lunch">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>+ добавить еду</string>
   </property>
  </widget>
  <widget class="QLabel" name="breakfast_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>Завтрак: 0 ккал</string>
   </property>
  </widget>
  <widget class="QLabel" name="lunch_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>Обед: 0 ккал</string>
   </property>
  </widget>
  <widget class="QLabel" name="app_logo_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poor Richard</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>MyDiet</string>
   </property>
  </widget>
  <widget class="QLabel" name="snack_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>Перекус: 0 ккал</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_to_breakfast">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>+ добавить еду</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_to_snack">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>+ добавить еду</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_to_dinner">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>+ добавить еду</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_food">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>430</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_food">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>390</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(185, 196, 201)</string>
   </property>
   <property name="text">
    <string>Сохранить изменения</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
